users.csv

```diff
@@ -0,0 +1,6 @@
+localusername,remoteusername,remotepassword,remotehostname,remoteport,security
+pansy.dumbledore5,user1@domain.com,"REPLACE_WITH_ACTUAL_PASSWORD",imap.domain.com,993,ssl
```

Comment:
Domains `example.com`, `example.org`, `example.net` are the RFC standard for examples. Please use it, always! https://www.rfc-editor.org/rfc/rfc2606.html

```suggestion
pansy.dumbledore5,user1@example.com,"REPLACE_WITH_ACTUAL_PASSWORD",imap.domain.com,993,ssl
```